npm/test.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+require('shelljs/global');
+require('colors');
+
+require('async').series([
+    require('./test-lint'),
+    require('./test-system'),
+    require('./test-integration'),
+    process.env.CI ? function (done) { done(); } : require('./test-browser')
```

Comment:
Where is test-unit? The browser tests are unit tests run in browser right? If so, we should take this call in that script. And allow to pass an argument `--browser` to force browser test locally. Two cents